corehq/apps/data_interfaces/models.py

```diff
@@ -114,13 +106,13 @@ def _get_case_property_value(current_case, name):
         def _add_update_property(name, value, current_case):
             while name.startswith('parent/'):
                 name = name[7:]
-                current_case = current_case.parent
+                # uses first parent if there are multiple
+                current_case = current_case.get_parent(identifier=DEFAULT_PARENT_IDENTIFIER)[0]
```

Comment:
this would raise an exception if there is no parent (should check that len() > 0)